### PR TITLE
Update auth-command according to new structure

### PR DESCRIPTION
**Pull request by kirtangajjar**
Depends on - https://github.com/EasyEngine/easyengine/pull/1220

### CODE REVIEW

File: src/Auth_Command.php [hunk 1]
```diff
@@ -52,75 +53,254 @@ public function __construct() {
...
+
+			$existing_auths = Auth::where( $query_conditions );
+
+			if ( ! empty( $existing_auths ) ) {
+				EE::error( $error_message );
+			}
+
+			$admin_tools_auth = Auth::get_global_admin_tools_auth();
+			if ( 'default' === $site_url && ! empty( $admin_tools_auth ) ) {
+					$user = $admin_tools_auth[0]->username;
```
Comment (mrrobot47) [inline src/Auth_Command.php]: Is this accidental? Also, check the line below.

File: src/Auth_Command.php [hunk 2]
```diff
@@ -52,75 +53,254 @@ public function __construct() {
...
+			if ( 'default' === $site_url ) {
+				$this->generate_global_auth_files();
+			} else {
+				$this->generate_site_auth_files( $site_url );
+			}
+
+			EE::log( 'Reloading global reverse proxy.' );
+			reload_global_nginx_proxy();
+
+			EE::success( sprintf( 'Auth successfully updated for `%s` scope. New values added/updated:', $this->site_data->site_url ) );
```
Comment (mrrobot47) [inline src/Auth_Command.php]: Change message to: 
```
Auth successfully added for `%s` scope. New values added:
```

File: src/Auth_Command.php [hunk 3]
```diff
@@ -52,75 +53,254 @@ public function __construct() {
...
+			EE::line( 'User: ' . $user );
+			EE::line( 'Pass: ' . $pass );
+		} else {
+			$file = EE_CONF_ROOT . '/nginx/vhost.d/' . $site_url . '_acl';
```
Comment (mrrobot47) [inline src/Auth_Command.php]: Remove this line as it is not used anywhere.

File: src/auth-utils.php [hunk 4]
```diff
@@ -14,27 +14,25 @@
...
-	$auth_data = [
-		'site_url' => 'default',
-		'username' => 'easyengine',
-		'scope'    => 'admin-tools',
-	];
-
-	if ( ! empty( Auth::where( $auth_data ) ) ) {
-		EE::log( 'Global auth exists on admin-tools. Use `ee auth list global` to view credentials.' );
+	if ( ! empty( Auth::get_global_admin_tools_auth() ) ) {
+		EE::log( 'Global auth exists on admin-tools. Use `ee auth list global_admin_tools` to view credentials.' );
```
Comment (mrrobot47) [inline src/auth-utils.php]: This log message seems wrong. Should it not be: __Use `ee auth list global` to view credentials.__

File: src/auth-utils.php [hunk 5]
```diff
@@ -14,27 +14,25 @@
...
+		'username' => 'easyengine',
+		'password' => $pass,
+	];
 
 	Auth::create( $auth_data );
-	EE::exec( sprintf( 'docker exec %s htpasswd -bc /etc/nginx/htpasswd/%s %s %s', EE_PROXY_TYPE, $site_auth_file_name, $auth_data['username'], $auth_data['password'] ) );
-	EE::success( sprintf( 'Global admin-tools auth added. Use `ee auth list global` to view credentials.' ) );
+
+	EE::exec( sprintf( 'docker exec %s htpasswd -bc /etc/nginx/htpasswd/default_admin_tools %s %s', EE_PROXY_TYPE, $auth_data['username'], $auth_data['password'] ) );
+	EE::success( sprintf( 'Global admin-tools auth added. Use `ee auth list global_admin_tools` to view credentials.' ) );
```
Comment (mrrobot47) [inline src/auth-utils.php]: Wrong log message again. Please check as mentioned above.

File: src/Auth_Command.php [hunk 6]
```diff
@@ -285,292 +562,22 @@ public function delete( $args, $assoc_args ) {
 	 *
 	 */
 	public function list( $args, $assoc_args ) {
+		//TODO: handle global_admin_tools
```
Comment (mrrobot47) [inline src/Auth_Command.php]: Remove this todo. `ee auth list global` handles it.

File: src/Auth_Command.php [hunk 7]
```diff
@@ -285,292 +562,22 @@ public function delete( $args, $assoc_args ) {
 	 *
 	 */
 	public function list( $args, $assoc_args ) {
```
Comment (mrrobot47) [inline src/Auth_Command.php]: Option to list and view whitelisted ips is missing. Handle it in `list` function with `--ip/whitelist` flag or something similar.

**Comment by kirtangajjar** [comment]
@mrrobot47 Done changes as requested.

**Pull request merged**